packages/frontend/editor-ui/src/components/MainHeader/WorkflowDetails.vue

```diff
@@ -684,28 +669,19 @@ const onBreadcrumbsItemSelected = (item: PathItem) => {
 							:class="$style['path-separator']"
 							>/</span
 						>
-						<ShortenName :name="name" :limit="value" :custom="true" test-id="workflow-name-input">
-							<template #default="{ shortenedName }">
-								<InlineTextEdit
-									:model-value="name"
-									:preview-value="shortenedName"
-									:is-edit-enabled="isNameEditEnabled"
-									:max-length="MAX_WORKFLOW_NAME_LENGTH"
-									:disabled="
-										readOnly || isArchived || (!isNewWorkflow && !workflowPermissions.update)
-									"
-									placeholder="Enter workflow name"
```

Comment:
Is removing the placeholder here intentional? If not, can you add it back and make sure it's not cut off by minWidth?

Reply:
Would be the correct way to do it, but since we only have English right now, It's alright.